meta-ostro/recipes-extended/libiio/libiio_git.bb

```diff
@@ -0,0 +1,14 @@
+DESCRIPTION = "Analog Devices Libiio"
+DEPENDS = "libxml2 bison flex avahi"
+LICENSE = "LGPLv2+"
```

Comment:
If it is LGPLv2.1, then say so here.

If openlmi-providers_0.6.0.bb got it wrong, then that's no reason to repeat the mistake here.